docs/build/apps/stellar-wallets-kit.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 🚧 Stellar Wallets Kit
+sidebar_position: 80
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
```

Comment:
I think we should start with `npm install` the package

Reply:
Made the required changes here https://github.com/od-hunter/stellar-docs/commit/aac521362750cedecd894e7ac1892cc92743bd95 . Kindly review